website/docs/for-iOS/quick-start.md

```diff
@@ -207,10 +217,19 @@ Now that the application has been configured all we need to do is add the Portal
 
 ```swift
 class ViewController: UIViewController {
-    override func loadView() {
-        self.view = PortalUIView(portal: "featured_products")
-    }
+  override func loadView() {
+    self.view = PortalUIView(portal: .featured_products)
```

Comment:
This should be updated to `.featuredProducts` when the extension definition of the Portal is updated to be camel case